src/float/printer/ryu_printf.cr

```diff
@@ -0,0 +1,628 @@
+{% skip_file if flag?(:wasm32) %}
```

Comment:
I think it would be helpful to have a local comment here about why it's skipped.  Perhaps copy the comment from https://github.com/crystal-lang/crystal/pull/14067/files#diff-4d36a71e2936361af3a9166fa7b7b97cbd2b0940b93d82fd1cf17d3eb9437aeeR310